app/src/main/java/chat/rocket/android/activity/MainActivity.java

```diff
@@ -172,6 +181,8 @@ public void showHome() {
   public void showRoom(String hostname, String roomId) {
     showFragment(RoomFragment.create(hostname, roomId));
     closeSidebarIfNeeded();
+    //TODO: Hide keyboard
+    KeyboardHelper.hideSoftKeyboard(this);
```

Comment:
Looks like might have a comment that needs removed.  The line right below seems to be doing exactly what the TODO calls for :)